### PR TITLE
Adds 4.8.21 release notes

### PR DESCRIPTION
**Pull request by darshan-nagaraj**
No BZ.

Not to be merged until 11-23.

Doc preview link: https://deploy-preview-39114--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-21

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -2887,3 +2887,26 @@ link:https://access.redhat.com/solutions/6507841[{product-title} 4.8.20 containe
...
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6532121[{product-title} 4.8.21 container image list]
+
+
+[id="ocp-4-8-21-bug-fixes"]
+==== Bug fixes
+
+This fix contains changes from Kubernetes 1.21.5. For more information, see link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#v1215[Kubernetes Changelog 1.21.5].
```
Comment (ctauchen) [inline release_notes/ocp-4-8-release-notes.adoc:2906]: Should this be feature instead? Is this different from what we did with new Kubernetes versions in the past? 

See https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-13

Comment (ctauchen) [reply release_notes/ocp-4-8-release-notes.adoc:2906]: If it is the same, might be worth using the same language as in previous versions.

**Review by ctauchen — COMMENTED** [review]
Question about whether the Kubernetes thing should be feature instead of a bug fix.

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (darshan-nagaraj) [reply release_notes/ocp-4-8-release-notes.adoc]: @ctauchen I have incorporated the changes suggested by Steven in our release-notes channel. PTAL

**Pull request merged**